models/topics.models.js

```diff
@@ -1,9 +1,13 @@
 const db = require('../db/connection')
-
+const endpoints = require('../endpoints.json')
 
 exports.getTopics = () => {
 return db.query('SELECT * FROM topics;')
 .then((result) => {
     return result.rows;
 })
+}
+
+exports.getEndpoints = () => {
+    return endpoints
```

Comment:
If you wanted you could just require this `endpoints` straight into the controller instead, rather than having a model function to return it